shared/src/components/tabulator/NullableInputEditor.vue

```diff
@@ -1,6 +1,20 @@
 <template>
   <div>
+    <date-picker
```

Comment:
I don't think we want to pollute the `NullableInputEditor` with the date picker.

Can we do this in a different way:

1. Add a named slot to the `NullableInputEditor` for the custom button
2. Pass in the calendar icon button into the slot
3. Then when that button is clicked it can show the date modal, this can then live a whole level UP from the nullableinputeditor.


Something like this: `DatePickerEditor.vue`

```vue
<NullableInputEditor ...>
  <template v-slot:buttons>
    <DatePicker ... />
  </template>
</NullableInputEditor

```